docker/conf/docker-compose-mac.yml.dist

```diff
@@ -13,9 +13,8 @@ services:
     webserver:
         image: nginx:1.13-alpine
         container_name: shopsys-framework-webserver
-        working_dir: /var/www/shopsys-framework
         volumes:
-            - shopsys-framework-sync:/var/www/shopsys-framework
+            - shopsys-framework-sync:/var/www/html
```

Comment:
I do not like this change. I cannot see any benefit (advantage while using shell ... you write sh and press Tab and you have it ...) 

The reason I do not like this is that our application is not HTML application but it is shopsys-framework application :)

Reply:
Let's use `shopsys` directory then.

Reply:
There will be still the `html` directory next to `shopsys` - that's a bit weird.

Reply:
We could rename it during the image build.

Reply:
The internal voting was 4:2 in favor of `/var/www/html`.